Web-App/templates/index.html

```diff
@@ -56,272 +52,259 @@
                       <span class="carousel-control-next-icon"></span>
                     </a>
                   </div>
-            </div>
-
-            <h1 class="display-4" style="margin-top:10px; font-size:40px; font-weight:bold" align="center">CricAI: Smart Outcome Prediction System</h1>
 
             <form method="post">
-                    <div class="row" style="margin-top:20px">
-                        <div class="col-md-4">

```

Comment:
Actually I am using bootstrap framework to handle the designing, `col-md-4` & `col-md-6` etc. are the bootstrap classes to handle Grids using bootstrap. No need to create custom classes for them